usdt-impl/src/stapsdt.rs

```diff
@@ -187,12 +189,96 @@ _.stapsdt.base:
     )
 }
 
+// Convert a supported data type to 1. a type to store for the duration of the
```

Comment:
is this copied from `common.rs`? why?

Reply:
Oop, I was under a mistaken impression that the common file isn't used in the stapsdt case and did a copy&paste. Fixed.